webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -54,14 +63,21 @@ const DEFAULT_MODAL_STATE = {
   modelName: '',
   tags: [],
   visible: false,
+  workspace: '1',
```

Comment:
Do we want the default modal state workspace to be `1` vs just `''` as is the default `useModalModelCreate`. I am fine with this it just seems like it could lead to unwanted behavior down the road of incorrectly choosing workspace 1.